index.js

```diff
@@ -18,7 +18,7 @@ module.exports = function arrayFirst(arr, num) {
   }
 
   var first = slice(arr, 0, isNumber(num) ? +num : 1);
-  if (+num === 1 || num == null) {
+  if (+num === 1 || num === undefined) {
```

Comment:
It's more rigorous than `num === undefined`.

```js
var undefined = 'foo';
var val;
console.log(val === undefined) // false
console.log(val == null) // true
```

This is because `undefined` can be a defined value, but `num == null` checks both `typeof num === 'undefined'` and `num === null`.